cmd/mcp-server/scenarios/resource-exhaustion-teardown.sh

```
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./common.sh was not specified as input (see shellcheck -x).

(SC1091)